nano/templates/ntnet_downloader.tmpl

```diff
@@ -129,4 +129,4 @@
 {{/if}}
 <br><br>
 
-<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.6 Copyright NanoTrasen 2450 - 2458</i></div>
\ No newline at end of file
+<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.6 Copyright NanoTrasen 2450 - 2568</i></div>
```

Comment:
I don't believe the current year is 2568.

Reply:
Thats when the copyright runs out

Reply:
@Lohikar I believe that this needs to be greater then current year at least by 5-10 years.

Reply:
The current year is 2460, btw. This is 108 years in the future, ICly.

Reply:
Ok and by extending the copyright law out, we never have to worry about this issue again.

Reply:
And it's invalid to future-date copyrights. IRL the copyright only can be updated when the relevant material is updated as well.